components/SummaryBox.vue

```diff
@@ -7,28 +7,24 @@
       <div class="usa-summary-box__text">
         <form class="usa-form">
           <fieldset class="usa-fieldset">
-            <template v-for="criteriaGroup in topLevelFilters">
-              <div
-                :id="'criteriaGroup-' + criteriaGroup.criteriaGroupKey"
-                :key="criteriaGroup.criteriaGroupKey">
-                <template v-for="criterion in getCriteriaMap(criteriaGroup.criteriaKey)">
-                  <h3
-                    id="summary-box-key-information"
-                    :key="`heading-${criterion.criteriaKey}`"
-                    class="usa-summary-box__heading">
-                    {{ criteriaGroup.label }}
-                  </h3>
-                  <CriteriaChild
-                    :key="criterion.criteriaKey"
-                    :criteria-key="criterion.criteriaKey"
-                    :label="criterion.label"
-                    :values="criterion.values.split('; ')"
-                    :type="criterion.type"
-                    :criteria-group-key="criteriaGroup.criteriaGroupKey"
-                    :response="criterion.response"
-                    class="margin-y-2 tablet:margin-y-3" />
-                </template>
-              </div>
+            <template v-for="filter in topLevelFilters">
+              <h3
+                id="summary-box-key-information"
+                :key="`heading-${filter.criteriaKey}`"
+                class="usa-summary-box__heading">
+                {{ filter.label }}
+              </h3>
+              <template v-for="criterion in getCriteriaMap(filter.criteriaKey)">
+                <CriteriaChild
```

Comment:
we should still look for getting this to load the one object instead of looping through an array of one value